### ISSUE TITLE
Automation for interacting with Tekton twitter account via gitops

### ISSUE THREAD

**Issue by bobcatfish** [tektoncd/community#39]
Okay so just like https://github.com/tektoncd/community/issues/38 but about our Twitter account instead!! https://twitter.com/tektoncd

At the moment only a few folks have access to the twitter account, and this means when there are announcements such as https://twitter.com/chmouel/status/1168821072441090048 they have to come from ppl's personal accounts (thanks @chmouel !) and get retweeted by the tektoncd account.

It would be very cool if folks working on Tekton could make new tweets by opening and merging pull requests.

Open question(s): not sure how to represent actions like "liking" and retweeting other random tweets. Also don't want to block folks from using the Twitter account directly? (Or it will probably never be used to retweet anything) - maybe something can be done to reconcile the state of the twitter account with the repo periodically?

**Comment by jlpettersson** [comment]
>It would be very cool if folks working on Tekton could make new tweets by opening and merging pull requests.

Yes, sounds very cool. And we could start with this functionallity.

>Open question(s): not sure how to represent actions like "liking" and retweeting other random tweets. Also don't want to block folks from using the Twitter account directly? (Or it will probably never be used to retweet anything) - maybe something can be done to reconcile the state of the twitter account with the repo periodically?

We can think about this and do that in other iterations?

**Comment by bobcatfish** [comment]
No one has gotten to this yet but I still think it's important to open up community resources such as the twitter account so that it's clear how to use them and more than just a select few have access

/lifecycle frozen
